### DISCUSSION TITLE
Have you considering leaving RSE?

### DISCUSSION THREAD

**Discussion by klapaukh** [RSEAA/RSEAA.github.io#123] [Discussion topics for #RSEAA24]
Have you considered leaving (or joining) the RSE profession? 
- What was pushing you to leave? 
    - If the status quo continues, do you think leaving is inevitable? 
- Why did you stay (or what was keeping you in)?
    - Have you had to sacrifice something to stay?  
- How are your answers different over the short-term vs long-term?

For those considering joining, the matching question is: what is it that attracts and worries you about the roles?

**Comment by rowlandm** [comment]
This is a very interesting topic Roma!
